src/extension.ts

```diff
@@ -73,7 +73,7 @@ export async function activate(context: vscode.ExtensionContext) {
 }
 
 function getServerOptions(context: vscode.ExtensionContext): languageclient.ServerOptions {
-    const languageServerPath = context.asAbsolutePath(path.join('node_modules', 'azure-pipelines-language-server', 'out', 'server', 'src', 'server.js'));
+    const languageServerPath = context.asAbsolutePath(path.join('node_modules', 'azure-pipelines-language-server', 'server.js'));
```

Comment:
Why did the path change here?

Reply:
The change to tsconfig.json [here](https://github.com/Microsoft/azure-pipelines-language-server/pull/5/files) got rid of the `server` part.  I'm not sure why `out` and `src` aren't needed anymore.